Users/abehenderson/CSX/Recursion/repeater.js

```diff
@@ -0,0 +1,31 @@
+// Attempt #2 (CSX Solution)
+
+function repeater(char) {
```

Comment:
This line contains the function signature for the parameter `char`: it is whatever is passed in.